test/public/eosReport/ecs-creation.test.js

```diff
@@ -124,11 +123,6 @@ module.exports = () => {
         await page.focus('#shift-flow .CodeMirror textarea');
         await page.keyboard.type('Shift flow\nOn multiple lines');
 
-        await page.waitForSelector('#from-previous-shifter .CodeMirror textarea');
-        expectInnerText(page, '#from-previous-shifter .CodeMirror textarea', info);
-        await page.focus('#from-previous-shifter .CodeMirror textarea');
-        await page.keyboard.type('Old information: ');
```

Comment:
Here you deleted too much. You still expect the info to be pre-filled! It just should be a markdown display